Header header

uint8 ros_state 
uint8 fault_code 
bool stack_faults 
bool raptorHeartfail 
uint8 counter 
bool brake_enable 
bool logging_enable
bool cancel_ros 


# stack_crash        1
# mission_cancelled  2
# ros_mode_disabled  3
# raptorHeartbeartfail 254
